src/Essentials/src/DeviceInfo/DeviceInfo.ios.tvos.watchos.cs

```diff
@@ -33,8 +33,10 @@ static string GetModel()
 		static string GetVersionString() => UIDevice.CurrentDevice.SystemVersion;
 
 		static DevicePlatform GetPlatform() =>
-#if __IOS__
+#if __IOS__ || IOS
 			DevicePlatform.iOS;
+#elif __MACCATALYST__ || MACCATALYST
```

Comment:
We may have to swap this because maccat is also ios, so we may have to put maccat first.